.env.example

```diff
@@ -0,0 +1,3 @@
+CONSENSUS_RPC=https://www.lightclientdata.org
+MAINNET_RPC=https://eth-mainnet.g.alchemy.com/v2/XXXXX
```

Comment:
Should we differentiate between rpc types?

Wouldn't it make more sense to have:
MAINNET_CONSENSUS_RPC
MAINNET_EXECUTION_RPC
GOERLI_CONSENSUS_RPC
GOERLI_EXECUTION_RPC

Reply:
Done @ncitron